.github/workflows/pull-request-ci.yaml

```diff
@@ -196,7 +196,7 @@ jobs:
           DEBUG_MODE: true
           GKE_NODE_COUNT: 1
           GKE_PREEMPTIBLE: false
-          GKE_INSTANCE_TYPE: n2d-highcpu-16
+          GKE_INSTANCE_TYPE: n2d-highcpu-32
```

Comment:
Note that on Concourse right now we are using 16, we may want to hold this change:

https://github.com/cloudfoundry-incubator/kubecf/blob/1986be55ff3957b94373eb25d934a9c2c80665d9/.concourse/pipeline.yaml.gomplate#L186

Reply:
Yep; at a minimum, we need more than 16 gigs of RAM (or rather, at least switch to `n2d`, which gives a full 16G rather than `n1` which gives slightly less), due purely due to our requests (it wasn't enough to start CATS).

I only bumped it to 32 because riding _that_ close to the edge seemed bad.

I'm happy to drop the change here to just `n2d`, if that helps; they're all in separate commits.